### ISSUE TITLE
GUI language selector _severly_ broken

### ISSUE THREAD

**Issue by rootfather** [mgba-emu/mgba#776]
The GUI language selector seems to be broken. On my Windows system, the menu entries in the main mGBA window are translated correctly, but each and every other dialog in mGBA always is displayed in english, no matter what language I choose in the Interface options.

After clicking "OK" and/or "Apply" and restarting mGBA, the main mGBA keeps the language I selected, but all other dialogs are still in english. It should be noted that after restarting mGBA, the language option in the "Interface" tab always falls back to English, no matter what I previously selected.

OS: Windows 10
mGBA revision da50382cd42765d48451fdf1f2fa608d87813626, compiled by myself

![mgba language bug](https://user-images.githubusercontent.com/11882577/27795090-022d4eac-6005-11e7-94c6-40e447a47498.PNG)

**Comment by endrift** [comment]
(Partially?) fixed by db08a75d9b031c3bfb6f510f6f52db6ace05c93e.

**Comment by rootfather** [comment]
Sadly, the fix doesn't work for me. The language= entry in qt.ini will not be created. Manually adding it doesn't help either. As before, the main menu items are translated, the rest of the UI is still English.

Compared to previous commits, this time the pulldown menu seems to remember the language you selected if you do this manually by setting language= in qt.ini. Again, even setting it manually doesn't help with the dialogs being untranslated.

**Comment by endrift** [comment]
Not sure what's going on with the dialogs, but it's supposed to use the default locale of your computer the first time you run it. Also, if you don't change the locale from your default, it's not supposed to make the key. I deleted the language key and set my Windows 10 machine to German (de_DE specifically) and when I started it I got the de translation. Setting it to English set it to the default en. Deleting the key again and setting it to de didn't set the key. So everything seems to work fine on my end apart from the dialogs.

Also some of the dialogs are translated (e.g. the log view), but many are not and I have no idea what's going on there.
